stable/ambassador/templates/crd/filterpolicy.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.pro.enabled -}}
```

Comment:
should these check if crds are enabled or are crds a requirement with pro? just for consistency!
I think expecting crds are fine as well :D ppl should be used to them :P

Reply:
Unfortunately, pro cannot watch annotations for configuration so CRDs are required.